src/main/app/src/app/shared/dynamic-table/model/zoeken-column.ts

```diff
@@ -0,0 +1,27 @@
+export enum ZoekenColumn {
+    BEHANDELAAR = 'behandelaar',
+    CREATIEDATUM = 'creatiedatum',
+    DAGEN_TOT_FATALEDATUM = 'dagenTotFataledatum',
+    DAGEN_TOT_STREEFDATUM = 'dagenTotStreefdatum',
+    EINDDATUM = 'einddatum',
+    EINDDATUM_GEPLAND = 'einddatumGepland',
+    GROEP = 'groep',
+    INDICATIES = 'indicaties',
+    NAAM = 'naam',
+    OMSCHRIJVING = 'omschrijving',
+    OPENSTAANDE_TAKEN = 'openstaandeTaken',
+    RESULTAAT = 'resultaat',
+    SELECT = 'select',
+    STARTDATUM = 'startdatum',
+    STATUS = 'status',
+    STREEFDATUM = 'streefdatum',
+    TOELICHTING = 'toelichting',
+    UITERLIJKE_EINDDATUM_AFDOENING = 'uiterlijkeEinddatumAfdoening',
+    URL = 'url',
+    ZAAK_IDENTIFICATIE = 'zaakIdentificatie',
+    ZAAK_IDENTIFICATIE2 = 'zaak.identificatie',
```

Comment:
ZAAK_IDENTIFICATIE2 hernoemen naar ZAAK_DOT_IDENTIFICATIE? Is maar een suggestie